presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -103,6 +103,9 @@ public Integer call()
         @Option(names = "--test-jar", paramLabel = "<jar>", description = "Path to test JAR " + DEFAULT_VALUE, defaultValue = "${product-tests.module}/target/${product-tests.module}-${project.version}-executable.jar")
         public File testJar;
 
+        @Option(names = "--cli-jar", paramLabel = "<jar>", description = "Path to executable cli JAR " + DEFAULT_VALUE, defaultValue = "${cli.bin}")
```

Comment:
`--cli-jar` -> `--cli-bin`, or `--cli-executable`?

"Path to executable cli JAR" -> "Path to CLI executable"

Reply:
Sure.